packages/launchpad/src/checklist-item/index.tsx

```diff
@@ -17,6 +17,10 @@ const ChecklistItem = ( { task, isPrimaryAction }: { task: Task; isPrimaryAction
 		actionDispatch && actionDispatch();
 	};
 
+	// Display task counter if task is incomplete and has the count properties;
+	const shouldDisplayTaskCounter =
+		! completed && task.target_repetitions && null !== task.repetition_count;
+
```

Comment:
Should we not show the task as complete with the `n/n` still displayed? I am not sure if we confirmed whether that should be the case.
Also, I think we should guard against `task.repetition_count` being `undefined`.
```suggestion
	// Display task counter if task is incomplete and has the count properties;
	const shouldDisplayTaskCounter = task.target_repetitions && ( null !== task.repetition_count && undefined !== task.repetition_count );

```